math/minuit2/src/MnTraceObject.cxx

```diff
@@ -18,32 +18,34 @@ namespace ROOT {
    namespace Minuit2 {
 
       void MnTraceObject::operator() (int iter, const MinimumState & state) {
+         MnPrint print("MnTraceObject");
 
-         MnPrint::PrintState(std::cout, state, "iteration  #  ",iter);
+         print.Debug(MnPrint::Oneline(state, iter));
```

Comment:
Actually this functionality (print in one line) can be used above in MnSeedGenerator.cxx when printing the initial state

Reply:
Right, I introduced this `Oneline` class to replace `PrintState`, since I want all printing to go through `Info`, `Debug` etc. Ok, I will use that above.